package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "restaurant",
+  "version": "1.0.0",
+  "description": "A restaurant page built with js and webpack",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --progress",
+    "watch": "webpack --progress --watch"
```

Comment:
### Optional Change
- [ ] How about we add a start script for faster local `web-dev server`? It will relieve our users of the code the hassle of running our code locally